air/src/stack/field_ops/mod.rs

```diff
@@ -214,23 +213,21 @@ pub fn enforce_and_constraints<E: FieldElement>(
     let b = frame.stack_item(1);
     let c = frame.stack_item_next(0);
 
-    // Enforce that a and b are binary values.
-    result[0] = op_flag * is_binary(a);
-    result[1] = op_flag * is_binary(b);
+    // Enforce that b is a binary.
```

Comment:
Maybe worth leaving a note in the comment to say that `is_binary` constraint for `a` is enforced as a general constraint (here an in other similar places).